docs/monitors/collectd-cassandra.md

```diff
@@ -60,7 +60,7 @@ The **nested** `values` config object has the following fields:
 
 The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
```

Comment:
If you're going to change the table header, then you have to change the text from:

Metrics that are marked as Included

to 

Metrics that are marked as "Default (non-custom)" 

throughout

I added quotes because now it's two words.

Reply:
I changed it.